src/lib/PipeSource.ts

```diff
@@ -15,8 +15,7 @@ export default class PipeSource extends EventEmitter {
 	}
 
 	protected publish(data: any): void {
-		console.debug('publish', data);
-		this.destinations.forEach(destination => {
+		this.destinations.forEach((destination) => {
```

Comment:
why did you add the parens? they don't improve readability and aren't required by the linter.